RocketRL/python/main.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun  6 11:49:31 2018
+
+This code is for an RL agent that determines the optimal input parameters for 
+the desired ouput for a 3d-printed rocket engine component. 
+
+Stage 0: load and visualize data, research past approaches to similar problems
+
+Stage 1: Implement an offline agent to learn optimal parameters from simulation data, 
+using shallow RL
+
+Stage 2: Above with keras-rl DQN and deep SARSA
+
+Stage 3: Bootstrap above with an engineer's inputs for parameters to test
+
+Stage 4: RL agent learns the optimal input parameters
+
+Stage 5: Parallelize and asynchonize the above
+
+
+Final Stage: what should ISP (measure of thrust) look like for a particular problem
+
+
+@author: ninalopatina
+"""
+import yaml
+import os
+
+#set home directory
+home_dir = '/Users/ninalopatina/gitrepos/Rocket_RL/'
+config_dir = 'config/'
```

Comment:
Same as previous comments, remove `home_dir` dependency and make `config_dir` a relative path.

One option would be to use the `os.getcwd()` command and assign it to `home_dir`. It will be a little more complicated in your case because of the nested structure you decided to use in your folders, but it's doable.

Another option is to use `argparse` to parse the arguments you pass to `main.py` when you call it from the terminal and pass the path to the config file when you run your program.

I have used both approaches in my insight project, you can see it on the two links below:
https://github.com/bguisard/davis/blob/davis/object_detection_app.py#L15
https://github.com/bguisard/davis/blob/davis/object_detection_app.py#L77

You could do something like:
```python
if __name__ == '__main__':
    parser = argparse.ArgumentParser()
    parser.add_argument('-c', '--config', dest='config_file', type=str,
                        default='../../config/config.yml', help='path to YAML config file')
    args = parser.parse_args()

    model_details = json.load(open(args.model_config))
    with open(args.config_file, 'r') as ymlfile:
        cfg = yaml.load(ymlfile)

```

In that case you will also need to move all of the important part of your code inside this new block, leaving just the function definitions and other global variables outside of the `__main__`. You can mirror what I did in my project if you are in doubt.